### PR TITLE
Add enrichment to split a resource by its providedLabel

### PR DESCRIPTION
**Pull request by no-reply**
This is needed to split the UW subjects.

### CODE REVIEW

File: lib/krikri/enrichments/split_provided_label_at_delimiter.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      resource.providedLabel = providedLabel
+      resource
+    end
+
+    ##
+    # @param value [#providedLabel]
+    # @return [Array<String>] a flat array of provided labels split from
+    #   the original
+    def split_value(value)
+      value.providedLabel.map { |l| l.split(';').map(&:strip) }.flatten
```
Comment (AudreyAltman) [inline lib/krikri/enrichments/split_provided_label_at_delimiter.rb]: Should this split by `@delimiter` instead of `;`?

Comment (no-reply) [inline lib/krikri/enrichments/split_provided_label_at_delimiter.rb]: yes!

**Comment by AudreyAltman** [comment]
Aside from the one comment above, this looks good to me.  It may be good for @anarchivist or @markbreedlove to take a look too, since they are more familiar with enrichments.

**Comment by anarchivist** [comment]
:+1:

**Pull request merged**